### PR TITLE
Audit script for prod dependencies

### PR DESCRIPTION
**Pull request by vladbalanescu**
Changes:
---

- Added a new audit script for spotting vulnerabilities through our prod deps

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +21,7 @@
     "lint": "yarn lint:es && yarn lint:style",
     "lint:es": "eslint 'packages/*/src/**/*.js'",
     "lint:style": "stylelint 'packages/*/src/**/*.scss' --report-needless-disables --report-invalid-scope-disables",
-    "test": "jest",
+    "test": "yarn run audit && jest",
```
Comment (SimonFinney) [inline package.json:24]: I'm wondering what your thoughts on keeping the audit and test scripts separate in `package.json`, and instead having a step specifically for auditing in CI? https://github.com/carbon-design-system/ibm-cloud-paks/blob/master/.github/workflows/ci-check.yml#L12

Comment (vladbalanescu) [reply package.json:24]: hmmm, yeah, I don't mind that either. I'll factor it in, cheers for the proposal 👍

**Comment by vladbalanescu** [comment]
@SimonFinney @asfordmatt Can you guys please have another look at my audit script, seems fine now 👍

**Review by SimonFinney — APPROVED** [review]
LGTM! Thank you 💯

**Pull request merged**